ui/packages/consul-ui/app/routes/dc/services/show.js

```diff
@@ -16,7 +16,7 @@ export default Route.extend({
       items: this.data.source(
         uri => uri`/${nspace}/${dc}/service-instances/for-service/${params.name}`
       ),
-      urls: this.settings.findBySlug('urls'),
+      urls: this.config.get().dashboard_url_template,
```

Comment:
I think this is fine for now as opposed to returning a promise, how come you changed you mind about this?

Reply:
Mostly because it felt like an extra step 🤔